core/typescript37Action/Dockerfile

```diff
@@ -0,0 +1,59 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM golang:1.12 as builder
+RUN env CGO_ENABLED=0 go get github.com/apache/openwhisk-runtime-go/main \
+    && mv /go/bin/main /bin/proxy
+FROM node:12.1.0-stretch
+ENV TYPESCRIPT_VERSION=3.7.4
+COPY --from=builder /bin/proxy /bin/proxy
+RUN apt-get update && apt-get install -y \
+    imagemagick \
+    graphicsmagick \
+    unzip \
+    wget \
+    && rm -rf /var/lib/apt/lists/* &&\
+    mkdir -p /app/action
+RUN cd /app ;\
+  npm install -g yarn ;\
+  npm install -g typescript@${TYPESCRIPT_VERSION} ;\
+  echo '{"private":true}' >package.json ;\
+  npm install --save --no-package-lock --production \
```

Comment:
need to remove all these packages with the exception of these packages which are installed in our node runtimes:
```
    "openwhisk": "3.21.1",
    "redis": "2.8.0",
    "uuid": "3.3.0"
```